nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/operation/HotReloadStudioSnapshot.java

```diff
@@ -127,9 +128,17 @@ public Blob run() throws Exception {
         }
     }
 
+    protected boolean shouldValidate() {
+        ConfigurationService cs = Framework.getService(ConfigurationService.class);
+        if (cs.isBooleanPropertyTrue("studio.snapshot.disablePkgValidation")) {
```

Comment:
It could be useful to factor out this constant, which is also used by `org.nuxeo.connect.client.jsf.AppCenterViewsManager.shouldValidateStudioSnapshot`

Reply:
agreed, any idea about where to put it?

Reply:
No 😀